client/rpcserver/types.go

```diff
@@ -36,6 +40,14 @@ type preRegisterResponse struct {
 	Fee uint64 `json:"fee"`
 }
 
+// RegisterForm is information necessary to register an account on a DEX.
+type RegisterForm struct {
+	URL     string `json:"url"`
+	AppPass string `json:"appPass"`
+	Fee     uint64 `json:"fee"`
+	Cert    string `json:"cert"`
+}
```

Comment:
I don't mind adding json tags to exported **core** types like `core.Registration` if that makes things easier.

Reply:
I also like the "Form" convention that has started. Can I change the name too?